src/test/java/io/openliberty/tools/intellij/it/SingleModMPProjectTestCommon.java

```diff
@@ -1007,6 +1020,67 @@ public static void deleteDirectoryIfExists(String dirPath) {
         }
     }
 
+    /**
+     * Clean project.
+     */
+    public void stopTerminal() {
+        if (!shouldCleanupTerminal) {
+            return;
+        }
+
+        Keyboard keyboard = new Keyboard(remoteRobot);
+        ProjectFrameFixture projectFrame = remoteRobot.find(ProjectFrameFixture.class, Duration.ofSeconds(10));
+        ComponentFixture terminal = remoteRobot.find(ComponentFixture.class, byXpath("//div[@class='JBTerminalPanel']"), Duration.ofSeconds(10));
+
+        terminal.rightClick();
+        ComponentFixture openFixtureNewTab = projectFrame.getActionMenuItem("New Tab");
+        openFixtureNewTab.click(new Point());
+
+        // Perform clean
+        String projectName = getSmMPProjectName();
+        if ("singleModMavenMP".equalsIgnoreCase(projectName)) {
+            keyboard.enterText("mvn liberty:stop");
+
+        } else if ("singleModGradleMP".equalsIgnoreCase(projectName) || "singleMod GradleMP".equalsIgnoreCase(projectName)) {
+            keyboard.enterText("gradle libertyStop");
+        }
+        keyboard.enter();
+        TestUtils.sleepAndIgnoreException(5);
```

Comment:
Waiting for server stop may need to be longer on Mac which is quite slow.

Reply:
Increased to 10s